### DISCUSSION TITLE
StatsForecast Plot height

### DISCUSSION THREAD

**Discussion by rsangole** [Nixtla/statsforecast#516] [General]
Hello,

Is there a way to change the default plotly output height for a `StatsForecast` object?

Cheers,
Rahul

**Comment by akmalsoliev** [comment]
> Hello,
> 
> Is there a way to change the default plotly output height for a `StatsForecast` object?
> 
> Cheers, Rahul

Hey Rahul, I guess I'm quite late to the party 😆. There is a way, however, it is not native to statsforecast. You would need to encapsulate your plot and then modify it using plotly. 

```py
fig = sf.plot(df)
fig.update_layout(your_settings)
```

for more information visit this website: 
https://plotly.com/python/setting-graph-size/
api: https://plotly.com/python/reference/layout/
